docs/CHANGELOG.asciidoc

```diff
@@ -38,15 +38,478 @@ Review the following information about the {kib} 8.3.0 release.
 
 [float]
 [[breaking-changes-8.3.0]]
-=== Breaking changes
+=== Breaking change
 
 Breaking changes can prevent your application from optimal operation and performance.
-Before you upgrade, review the breaking changes, then mitigate the impact to your application.
+Before you upgrade, review the breaking change, then mitigate the impact to your application.
+
+// tag::notable-breaking-changes[]
+
+[discrete]
+[[breaking-129581]]
+.Removes Quandl and Graphite integrations
+[%collapsible]
+====
+*Details* +
+The experimental `.quandl` and `.graphite` functions and advanced settings are removed from *Timelion*. For more information, check {kibana-pull}129581[#129581].
+
+*Impact* +
+When you use the `vis_type_timelion.graphiteUrls` kibana.yml setting, {kib} successfully starts, but logs a `[WARN ][config.deprecation] You no longer need to configure "vis_type_timelion.graphiteUrls".` warning.
+
+To leave your feedback about the removal of `.quandl` and `.graphite`, go to the link:https://discuss.elastic.co/c/elastic-stack/kibana/7[discuss forum].
+====
+      
+// end::notable-breaking-changes[]
```

Comment:
```suggestion
[discrete]
[[breaking-134855]]
.Makes Osquery All with All base privilege
[%collapsible]
====
*Details* +
The Osquery {kib} privilege has been updated, so that when the *Privileges for all features level* is set to *All*, this now applies *All* to Osquery privileges as well. Previously, users had to choose the *Customize* option to grant any access to Osquery. For more information, refer to {kibana-pull}130523[#130523].

*Impact* +
This impacts user roles that have *Privileges for all features* set to *All*. After this update, users with this role will have access to the Osquery page in {kib}. However, to use the Osquery feature fully, these requirements remain the same: users also need Read access to the logs-osquery_manager.result* index and the Osquery Manager integration must be deployed to Elastic Agents.
====

// end::notable-breaking-changes[]
```